### DISCUSSION TITLE
USB Audio Interface Input -> Output

### DISCUSSION THREAD

**Discussion by tubby38** [ExistentialAudio/BlackHole#792] [Q&A]
Hello,

I have a Behringer UCA202 USB interface to capture Xbox console audio to my Mac Mini M1 via the Astro TR Mixamp.  It can record audio in Audacity fine as it recognizes input, however, I want to use it to directly route to my Rodecaster Duo Secondary USB.  Routing other types of audio to the Rodecaster works...just can't get the USB audio interface to work as a proper input

Installed the 2ch. and 16ch. based on previous suggestions but is this not possible as it's similar to #747?

Xbox -> Astro Mixamp TR (Line Out, 3.5mm) -> Mac Mini, Behringer UCA202 (3.5mm Female to RCA In/USB Audio Codec) -> Blackhole (Aggregated Input)-> Rodecaster Duo Secondary (Output)

I also have a Behringer [XENYX 302USB](https://www.behringer.com/product.html?modelCode=0601-ADN) being delivered to try a different interface type and see if the mixing route would help.

![input](https://github.com/ExistentialAudio/BlackHole/assets/23019639/fac5c888-c7df-4720-b5d7-f1e0c13abaff)

**Comment by devinroth** [comment]
Maybe try our new app https://dipper.audio.

Devin
